src/main/java/seedu/address/logic/commands/AddSessionCommand.java

```diff
@@ -15,19 +18,24 @@
  */
 public class AddSessionCommand extends Command {
 
-    public static final String COMMAND_WORD = "Session";
+    public static final String COMMAND_WORD = CommandWords.SESSION + " " + CommandWords.ADD_MODEL;
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a session to the TATracker. "
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a session in TA-Tracker. "
```

Comment:
I think even the old message was fine. New one is also fine lol

Reply:
Oh its just for the hyphen!